src/commands/scale.ts

```diff
@@ -47,6 +47,7 @@ export default class Scale extends BaseCommand {
   }];
 
   async run(): Promise<void> {
+    this.log('This feature is in alpha. Please send any comments or questions to support@architect.io.');
```

Comment:
2 quick things.
1. Should be yellow
2. Should mention it is subject to change.
```
This feature is in alpha. While the feature should be stable, it may be changed or removed without prior notice. As such we do not recommend using this feature in any automated pipelines.

During this time we greatly appreciate any feedback as we continue to finalize the implementation. You can reach us at support@architect.io.
```
This will make sure
1. It is noticed since a giant yellow text string is usually not missed.
2. They understand we are striving to make it stable but guarantee nothing.
3. Set limitations on how we expect the feature to be used currently.
4. That we value their insight
5. That at some point this will move out of alpha

Reply:
Updated:

![image](https://user-images.githubusercontent.com/8558567/188949617-cd49720d-131d-4618-b575-dde9c11b8b34.png)